README.rst

```diff
@@ -81,9 +84,54 @@ Usage
 More documentation on using ``twine`` to upload packages to PyPI is in
 the `Python Packaging User Guide`_.
 
+Keyring Support
+---------------
+
+Instead of typing in your password every time you upload a distribution, Twine
+allows you to store your username and password securely using `keyring`_.
+
+To use the keyring, you must first install the keyring packages:
+
+- On Windows and MacOS you just need to install ``keyring``, for example,
+  ``pip install --user keyring``.
+- On Linux, in addition to the ``keyring`` package you also need to ensure the
+  ``python3-dbus`` system package is installed. For example, ``apt install
+  python3-dbus``. See `Keyring's installation instructions`_ for more details.
+
+Once keyring is installed you can use the ``keyring`` program to set your
+username and password for Twine.
+
+To set your username and password for test PyPI:
+
+.. code-block:: console
+
+    $ keyring set https://test.pypi.org/legacy/ your-username
+    # or
+    $ python3 -m keyring set https://test.pypi.org/legacy/ your-username
+
+To set your username and password for PyPI:
+
+.. code-block:: console
+
+    $ keyring set https://upload.pypi.org/legacy/ your-username
+    $ python3 -m keyring set https://upload.pypi.org/legacy/ your-username
```

Comment:
Please include the "or" between these two lines as you did above.

Reply:
Done.